icu4c/source/i18n/unicode/listformatter.h

```diff
@@ -234,12 +233,11 @@ class U_I18N_API ListFormatter : public UObject{
      * @param errorCode ICU error code returned here.
      * @return          Formatted string combining the elements of items,
      *                  appended to appendTo.
-     * @draft ICU 63
+     * @stable ICU 63
```

Comment:
Don't promote this, as discussed (superseded by formatToValue)

Reply:
got it